internal/internal_nexus_task_handler.go

```diff
@@ -200,51 +191,62 @@ func (h *nexusTaskHandler) handleStartOperation(
 	}
 	var opres nexus.HandlerStartOperationResult[any]
 	var err error
+	var panic bool
 	func() {
 		defer func() {
 			recovered := recover()
 			if recovered != nil {
+				panic = true
 				var ok bool
 				err, ok = recovered.(error)
 				if !ok {
 					err = fmt.Errorf("panic: %v", recovered)
 				}
 
-				nctx.Log.Error("Panic captured while handling nexus task", tagStackTrace, string(debug.Stack()), tagError, err)
+				nctx.Log.Error("Panic captured while handling Nexus task", tagStackTrace, string(debug.Stack()), tagError, err)
 			}
 		}()
 		opres, err = h.nexusHandler.StartOperation(ctx, req.GetService(), req.GetOperation(), input, startOptions)
 	}()
 	if ctx.Err() != nil {
+		if !panic {
+			nctx.Log.Error("Context error while processing Nexus task", "error", ctx.Err())
+		}
 		return nil, nil, errNexusTaskTimeout
 	}
 	if err != nil {
+		if !panic {
+			nctx.Log.Error("Handler returned error while processing Nexus task", "error", err)
+		}
```

Comment:
```suggestion
		if !panic {
			nctx.Log.Error("Context error while processing Nexus task", tagError, ctx.Err())
		}
		return nil, nil, errNexusTaskTimeout
	}
	if err != nil {
		if !panic {
			nctx.Log.Error("Handler returned error while processing Nexus task", tagError, err)
		}
```

Or is it intentionally not using that tag?